types/k6/net/grpc.d.ts

```diff
@@ -29,7 +29,7 @@ export interface Params {
 
     tags?: object;
 
-    timeout?: string;
+    timeout?: string | number;
```

Comment:
Please, add a test for `string`.

Reply:
Sure, added a case for `string` type as well
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57368/commits/51d9ac5e4aefd73291f2259bf15bb0ecd959347b